netlink-packet-route/src/rtnl/route/message.rs

```diff
@@ -1,11 +1,12 @@
 use crate::{
     nlas::route::Nla,
     traits::{Emitable, Parseable},
-    DecodeError, RouteHeader, RouteMessageBuffer,
+    DecodeError, RouteHeader, RouteKind, RouteMessageBuffer, RouteProtocol, RouteScope, RouteTable,
```

Comment:
I was thinking of removing `RouteKind` and `RouteProtocol` and `RouteScope`, [just like I removed `NetlinkFlags`, `LinkFlags` and `LinkLayerType`](https://github.com/little-dude/netlink/commit/252822f8c72e497bfcbe7794eac963b089d9b7ad). This is unrelated to this PR but do you have an opinion about this?